User/src/UserService.js

```diff
@@ -62,16 +71,18 @@ const createUser = async (email, password, confirmPassword) => {
     });
 
     // Check if the password is at least 8 characters long
-    if (password.length < 8) {
+    if (password.length < MIN_PASSWORD_LENGTH) {
       throw Object.assign(
         new Error('Password must be at least 8 characters long'),
```

Comment:
Should we replace 8 with MIN_PASSWORD_LENGTH

Reply:
8 was replaced with MIN_PASSWORD_LENGTH, a constant, to avoid the use of magic numbers